app/assets/stylesheets/base/_buttons.scss

```diff
@@ -96,3 +96,9 @@ button.close {
   color: #fff;
 }
 
+.btn-delete-client-note {
```

Comment:
Let's see if we can use the built-in classes for this, maybe .btn.btn-default.btn-small and then wrap it in some 
.row 
  .col-sm-8
  .col-sm-4.text-right

Another option is just to use the icon for an X (%i.icon-cross)